ros_bt_py/ros_bt_py/tree_manager.py

```diff
@@ -576,49 +574,45 @@ def tick(self, once=None):
                 )
                 return
 
+        tick_start_timestamp = self.ros_node.get_clock().now()
         while True:
-            tick_start_timestamp = self.ros_node.get_clock().now()
```

Comment:
I am not sure if I understand this change? Why not get the new timestamp at the beginning of each iteration instead of before the loop and at the end of each iteration?